resources/controllers/Consumer/Subscriptions.php

```diff
@@ -68,14 +68,16 @@ public function checkout()
         global $stripe_pk;
         global $videoId;
 
-        if (isset($_GET['plan_id']) && $plan = \Zype::get_plan(filter_var($_GET['plan_id'], FILTER_SANITIZE_STRING))) {
-            $za           = new \ZypeMedia\Services\Auth;
-            $consumer_id  = $za->get_consumer_id();
+        $plan_id = $this->request->validate('plan_id', ['textfield']);
+
+        if ($plan_id && $plan = \Zype::get_plan($plan_id)) {
+            $za = new \ZypeMedia\Services\Auth();
+            $consumer_id = $za->get_consumer_id();
             $access_token = $za->get_access_token();
-            $consumer     = \Zype::get_consumer($consumer_id, $access_token);
+            $consumer = \Zype::get_consumer($consumer_id, $access_token);
```

Comment:
@Osoro Same validation stuff with `$plan_id` in [Line 114](https://github.com/zype/zype-wordpress-plugin/pull/38/files?utf8=%E2%9C%93&diff=split&w=1#diff-d8ea67f0f62209d7b0bed371f19bd7f8R114)